spec/unit/libraries/path_helpers_spec.rb

```diff
@@ -1,9 +1,11 @@
-require_relative 'spec_helper'
+require_relative '../spec_helper'
```

Comment:
minor, next time you're in here these can be changed to just `require 'spec_helper'`.  Then if they move around again it doesn't matter what directory they're in in relation to the spec helper file.